public/landing.html

```diff
@@ -28,6 +28,282 @@
+          </li>
+
+          <li>
+            <p class="navbar-btn">
+              <a href="#" class="btn btn-default">
+                Try Filebox Business
+              </a>
+            </p>
+          </li>
+        </ul>
+      </div>
+      </div><!-- /.navbar-collapse -->
+    </div><!-- /.container-fluid -->
+  </nav>
+
+  <div class="container padding-inside-top-md padding-inside-bottom-md">
+    <div class="row">
+      <div class="col-sm-12 col-md-offset-1 col-md-10">
+        <img src="assets/images/hero.png" alt="">
+      </div>
+
+      <div class="row">
+        <div class="col-md-offset-1 col-md-4">
+          <h1>Filebox works the way you do</h1>
+
+          <p>
+            Get to all your files from anywhere, on any device, and share them with anyone.
+          </p>
+        </div>
+
+        <div class="col-md-offset-2 col-md-4">
+          <form>
+            <div class="form-group">
+              <input type="text" class="form-control" id="exampleInputEmail1" placeholder="Full name">
+            </div>
+
+            <div class="form-group">
+              <input type="email" class="form-control" id="exampleInputEmail1" placeholder="Email">
+            </div>
+
+            <div class="form-group">
+              <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password">
+            </div>
+
+            <div class="checkbox">
+              <label>
+                <input type="checkbox"> I agree to <a href="#">Filebox terms</a>
+              </label>
+            </div>
+
+            <div class="form-group">
+              <button type="submit" class="btn btn-primary btn-block">Sign up for free</button>
+            </div>
+
+            <hr>
+
+            <div class="form-group">
+              <button type="submit" class="btn btn-primary btn-block"><i class="fa fa-google"></i> Sign up free with Google</button>
+            </div>
+          </form>
+        </div>
+      </div>
+
+      <div class="row padding-inside-top-lg"></div>
+      <div class="col-md-offset-1 col-md-4">
+        <h1>Take your docs anywhere</h1>
+
+        <p>
+          Save files on your computer, then access them on your phone from the road. Everything you keep in Filebox is synced automatically to all your devices.
+        </p>
+      </div>
+      <div class="col-md-offset-2 col-md-4">
+        <img src="assets/images/docs-anywhere.png" alt="">
+      </div>
+    </div>
+
+    <div class="row padding-inside-top-lg">
+      <div class="col-md-offset-1 col-md-4">
+        <img src="assets/images/send-videos-quickly.png" alt="">
+      </div>
+
+      <div class="col-md-offset-2 col-md-4">
+        <h1>Send videos quickly</h1>
+
+        <p>
+          Send your entire wedding video to family with a simple link. It’s easy to share large files with anyone — even if they don’t have a Filebox account.
+        </p>
+      </div>
+    </div>
+
+    <div class="row padding-inside-top-lg"></div>
```

Comment:
It looks like you may have accidentally closed your `<div class="row">` here? This means the `<div class="col-...">`s below are orphaned.